packages/internals/renderer/src/adapter.test.js

```diff
@@ -28,16 +30,16 @@ describe('Adapter', function () {
 
   describe('.match()', function () {
     it('tests if the file provided can be handled by the adapter', function () {
-      ['.fjk', ['.fjk'], (file => file.extname === '.fjk')].forEach(match => {
+      ['.fjk', ['.fjk'], (path => extname(path) === '.fjk')].forEach(match => {
         const adapter = new Adapter(Object.assign({}, validAdapter, {match}));
-        expect(adapter.match(funjucksFile)).to.equal(true);
-        expect(adapter.match(otherFile)).to.equal(false);
+        expect(adapter.match(funjucksFile.path)).to.equal(true);
+        expect(adapter.match(otherFile.path)).to.equal(false);
       });
     });
 
-    it('throws an error if the file argument is not a File instance', function () {
+    it('throws an error if the file argument is not a path', function () {
```

Comment:
Does it expect an actual path-like-string here, or just a string? Might be worth renaming for clarity if latter.